src/macrofier/engine.mjs

```diff
@@ -576,6 +635,7 @@ function generateSchemas(json, templates, options) {
     }    
 
     let content = getTemplate('/schemas/default', templates)
+    content = content.trim()
```

Comment:
It can cause undesired side-effects to trim whitespace from the content. For example Markdown is sometimes picky about how much whitespace is needed for a document to render properly.

Can we remove this?

Reply:
now updated logic to trim only there is only whitespaces, otherwise trimEnd of whitespace. It is added to avoided unwanted newlines